### PR TITLE
Refresh frames when dragging trimming handles

### PR DESCRIPTION
**Pull request by qbait**
Before (not refreshed):
before

After (refreshed)
after

### CODE REVIEW

File: lib/src/trim_editor.dart [hunk 1]
```diff
@@ -516,4 +520,13 @@ class _TrimEditorState extends State<TrimEditor> with TickerProviderStateMixin {
       ),
     );
   }
+
+  _refreshVideoIfNeeded() async {
+    if (Platform.isIOS) {
+      await videoPlayerController.play();
```
Comment (adamstyrc) [inline lib/src/trim_editor.dart:526]: Last question below wont work: ?
```
await videoPlayerController.play()
awair videoPlayerController.pause()
```

File: lib/src/trim_editor.dart [hunk 2]
```diff
@@ -516,4 +520,13 @@ class _TrimEditorState extends State<TrimEditor> with TickerProviderStateMixin {
...
+      Timer(_refreshVideoDuration, () {
```
Comment (LugonjaAleksandar) [inline lib/src/trim_editor.dart:527]: How did we come up with this magic 50ms number? What if on some devices it takes 51 or 52ms and this call fails?

File: lib/src/trim_editor.dart (hunk 1 above)
Comment (qbait) [reply lib/src/trim_editor.dart:526]: Nope, that was also my first try. Unfortunately, it won't work.

File: lib/src/trim_editor.dart (hunk 2 above)
Comment (qbait) [reply lib/src/trim_editor.dart:527]: It's empiric. The ideal would-be if `seekTo` in `video_player` is consistent on `iOS` and `Android` and properly moves to a certain frame, but it doesn't, so we need to apply this hack. If you have a better solution in mind, let's try it!
If for some reason this call fails, the behavior will be the same as in the old version (frame won't be refreshed).

**Review by Sauraus — APPROVED** [review]

**Pull request merged**